contracts/nois-sink/src/msg.rs

```diff
@@ -10,6 +10,7 @@ pub struct InstantiateMsg {}
 pub enum ExecuteMsg {
     /// Burns the tokens that are sent as `funds` with this message
     Burn {},
+    BurnNative {},
```

Comment:
There are both native coins. Maybe this is better?

```suggestion
    /// Burns all unois owned by the contract. This can be used
    /// if tokens were sent here using bank sends.
    BurnBalance {},
```